### PR TITLE
set up for draft-02 and fix typos

### PR DESCRIPTION
**Pull request by mamund**

### CODE REVIEW

File: draft-02.xml [hunk 1]
```diff
@@ -0,0 +1,1566 @@
...
+<!-- ref commands -->
+<?rfc comments="yes"?>
+<?rfc inline="yes"?>
+<?rfc rfcedstyle="yes"?>
+<?rfc toc="yes"?>
+<?rfc symrefs="yes"?>
+<?rfc compact="yes"?>
+<?rfc subcompact="no"?>
+<!-- the actual document! -->
+<rfc docName="draft-amundsen-richardson-foster-alps-01" ipr="trust200902">
```
Comment (fosrias) [inline draft-02.xml]: You want change this to 02?

**Comment by fosrias** [comment]
@mamund Only one issue I see. Update draft-02.xml to render 02 version and re-generate other files. Still on 01 per comment above. After that :+1: to merge.

**Comment by mamund** [comment]
derp!

i'll make the change, regen and commit then merge in.

we can do additional PRs against the -02 once we identify issues for the
milestone

cool?



mamund
+1.859.757.1449
skype: mca.amundsen
http://amundsen.com/blog/
http://twitter.com/mamund
https://github.com/mamund
http://linkedin.com/in/mamund

**Comment by fosrias** [comment]
Yup.

**Pull request merged**